tests/e2e/models/cosmos3/e2e_plugins/runners/diffusion.py

```
@@ -0,0 +1,230 @@
+        case: E2ECase,
+        ctx: RunContext,
+        bundle_path: str,
+    ) -> list[str] | None:
+        if case.task_strategy != self.strategy_name:
+            return None
+        env = {**os.environ, "LD_LIBRARY_PATH": ctx.ld_library_path}
+        _prepare_distributed_env(case, ctx, env)
+        command = build_generate_video_command(
+            case,
+            ctx,
+            Path("/tmp/trtmc_cosmos3_frames"),
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/trtmc_cosmos3_frames"

(S108)